mimic-navigation/navigation.md

```diff
@@ -0,0 +1,97 @@
+# Navigating Mimic
+
+Mimic is a single-window application. The interface is divided into four main parts:
+
+- **Action Bar**
+- **Routes**
+- **Logs**
+- **Status bar**
+
+Every part of the application was designed to give you enough space for showing information.
+
+## Action Bar
+
+![alt text](../Images/actionbar.png 'Action bar')
+
+On the Action Bar, the following options are available:
+
+- **Projects** Button - Create a new project, switch between projects.
+
+- **Edit** Icon - Edit the project name.
+
+- **Start Server** Button - Start the server, stop the server.
+
+- **Plus(+)** Button - Add a route to the project.
+
+- **Menu** Button - Projects settings, delete all routes, delete project.
+
+## Routes
+
+![alt text](../Images/routes.png 'Routes')
+
+The user can see a list of routes. Every route contains a description, method, type and endpoint path.
+
+### Routes Details
+
+When the user clicks on an item in the routes' list, the Side Bar is shown with details about the selected endpoint.
+
+### Side Bar - Endpoints Details
+
+![alt text](../Images/endpointdetails.png 'Endpoints Details')
+
+The Side Bar has four main parts:
+
+- **Endpoint Details Action Bar** - Settings for the endpoint.
+- **Path** - Input where the user can set an endpoint path.
+- **Request/Response** - Two buttons let the user switch between Request and Response options.
+- **Methods and Body** - Depends on the **Request/Response** buttons.
+
+#### Endpoint Details Action Bar
+
+![alt text](../Images/detailsoptions.png 'Endpoints Details')
+The Action Bar contains:
+
+- **Menu Button** - Allows the user to delete an endpoint.
+- **Save Icon** - Allows the user to save endpoint details, making it active after every change.
+- **Close Icon** - Closes the endpoint details.
+
+#### Path
+
+An user can type the endpoint path and save it using _Enter_ or _Save Icon_.
```

Comment:
[style] _An user_ -> _you_

The documentation is written for the user, so we can address them directly. I think it's easier to understand this way. You can also check the other instances of _user_ that you have in the docs to check if those could be improved.